README.md

```diff
@@ -1,153 +1,96 @@
-- Create shortcuts for the tasks in your `keybindings.json` file. You can access this file by opening the keyboard shortcuts pane (preferences -> keyboard shortcuts), and then hitting the little button in the upper right corner labeled Open Keyboard Shortcuts (JSON). Then add entries like the following, where `args` matches the Task's `label`:
-```json
-[
-    {
-        "key": "shift+enter",
-        "command": "workbench.action.tasks.runTask",
-        "args": "Jupyter Ascending Run Cell"
-      },
-      {
-        "key": "alt+s",
-        "command": "workbench.action.tasks.runTask",
-        "args": "Jupyter Ascending Sync"
-      }
-]
-```
-
-Then everything should work properly! If you see error messages, open the Terminal (View -> Terminal) and run the Task again to see the command output. It should also have the path to the full log file with more detailed logs.
-
-If you'd like to auto-sync on file save, you can try an extension like [Trigger Task on Save](https://marketplace.visualstudio.com/items?itemName=Gruntfuggly.triggertaskonsave), with a file filter to only sync `.sync.py` files.
-
-
-Things we'd like help improving:
-- The terminal always gets opened when running a task, even with "presentation":"reveal":"never" set in the Task. This seems to be due to an issue with the VSCode Python extension (see [here](https://github.com/microsoft/vscode/issues/65179)). Just making the window small is probably the best workaround for now.
+Currently Jupyter Ascending expects the jupyter server to be running at `localhost:8888`. If it's running elsewhere (eg due to having multiple jupyter notebooks open), you'll need to set the env variables `JUPYTER_ASCENDING_EXECUTE_HOST` and `JUPYTER_ASCENDING_EXECUTE_PORT` appropriately both where you use the client (ie in your editor) and where you start the server.
 
+By default the Jupyter server will search for a free port starting at 8888. If 8888 is unavailable and it selects eg 8889, Jupyter Ascending won't work - as it's expecting to connect to 8888. To force Jupyter to use a specific port, start your jupyter notebook with `JUPYTER_PORT=8888 JUPYTER_PORT_RETRIES=0 jupyter notebook` (or whatever port you want, setting also `JUPYTER_ASCENDING_EXECUTE_PORT` appropriately).
 
-## Usage in other editors
+### Working on a remote server
 
-To use Jupyter Ascending in a different editor, you just need to be able to set up a few hooks:
+Jupyter Ascending doesn't know or care if the editor and the jupyter server are on the same machine. The client is just sending requests to `http://[jupyter_server_url]:[jupyter_server_port]/jupyter_ascending`, with the default set to `http://localhost:8888/jupyter_ascending`. We typically use SSH to forward the remote jupyter port into `localhost:8888`, but you can set up the networking however you like, and use the environment variables to tell the client where to look for the Jupyter server.
```

Comment:
add line first in this section: install on both the client and server

Reply:
also provide an example SSH forwarding command, no need for people to figure it out from scratch!